### DISCUSSION TITLE
Time for a new release?

### DISCUSSION THREAD

**Discussion by canton7** [nathanmarlor/foxess_modbus#152] [General]
Hey @nathanmarlor,

This integration is starting to gain some popularity (at least, according to engagement on the facebook group), and we've got quite a few bug fixes which I think it would be good to get released, before too many people start running into them.

(We've also got people running off the latest `main`, because even the betas are too old to contain the features they want to use).

Let me know if you'd rather I merged my own PRs, and if you're happy for me to cut releases. I don't want to just cruise in and start taking over....

Thanks!

**Comment by canton7** [comment]
(I've also got quite a large change to the config flow coming up, and I'd like to give that some time in the betas before releasing, but I don't want it to hold up everything else...)

**Comment by nathanmarlor** [comment]
@canton7 agreed, apologies for going AWOL - been away this week. I'll find time Monday latest to catch up on everything and drop a release (+ FB update in the FoxESS group).

**Reply by canton7** [reply]
No worries! Hope you had a good time away 🏖️

**Comment by nathanmarlor** [comment]
Done - v1.4.0 out in the wild :rocket:

**Reply by canton7** [reply]
Yay, thank you!

It occurred to me too late that we should probably list the new services in the README... Oh well. We probably want to move those to the wiki anyway?

**Reply by nathanmarlor** [reply]
I'll update the readme today and do a sneaky push to main :shipit:
